domain/src/test/java/org/oppia/android/domain/exploration/ExplorationProgressControllerTest.kt

```diff
@@ -1986,8 +1990,8 @@ class ExplorationProgressControllerTest {
     waitForGetCurrentStateSuccessfulLoad()
 
     // Resuming shouldn't log a 'start card' event.
-    val eventLog = fakeAnalyticsEventLogger.getMostRecentEvent()
-    assertThat(fakeAnalyticsEventLogger.getEventListCount()).isEqualTo(1)
+    val eventLog = fakeAnalyticsEventLogger.getOldestEvent()
+    assertThat(fakeAnalyticsEventLogger.getEventListCount()).isEqualTo(4)
```

Comment:
Why are an additional 3 events being logged on resume?

Reply:
The additional 3 events logged are progress saving success events. The event is logged three times because the checkpoint is saved 3 times in `beginExplorationImpl` at line 570 [here](https://github.com/oppia/oppia-android/blob/9e8553cdcebb5f40b170d33b2e02bbe02081530b/domain/src/main/java/org/oppia/android/domain/exploration/ExplorationProgressController.kt#L565-L572).